### ISSUE TITLE
'Same shipping address' setting not remembered on future Checkouts.

### ISSUE THREAD

**Issue by meredevelopment** [ingenesis/shopp#3554]
**Setup:** 
If a customer checks-out, un-ticks the "Same shipping address" checkbox, then adds a different Shipping Address, both addresses are recorded against the purchase and the customer record when checkout is completed. good.

**Problem:**
On MacOS Chrome and Safari, when the customer returns to checkout again, the "Same shipping address" is ticked again. If they un-tick it, the alternative shipping address they set previously is still visible. If they do not un-tick it, and complete the checkout, the shipping address for the order and the customer address gets set to the billing address, overwriting their previously set shipping address.

**What I expect to see:** 
When 'Same shipping address' is ticked or un-ticked, that 'state' should be saved and respected for all future orders unless changed by the customer.

**Notes:** 
On Firefox (Mac OS), the behaviour is as expected.

**Comment by meredevelopment** [comment]
@sierrab123 When you get a chance, can you test this one 1.3.11 Windows / IE & Chrome please?

**Comment by meredevelopment** [comment]
This has been confirmed by @sierrab123 at the time. Needs a fix for acceptable UX.

**Comment by Just-m** [comment]
I don't understand what the problem is.
Let's say I order something for my brother, enter my billingaddress and his shippingaddress.
This is stored in my account.
Next time I order something for myself. Now you want to keep my brother's address as well?

**Comment by meredevelopment** [comment]
@Just-m yes in your example, I agree it's not a big deal. However what's happening on my client's store is that a customer will frequently set a shipping address as their workplace, as the are never home to sign-for deliveries. The problems they (and I) have with this issue are: 

- firstly that the checkbox state isn't remembered for logged-in users. It's the sort of thing one expects to be remembered when they have an account. 
- Secondly, the overwriting of the previously set shipping address. 

Maybe it's just a matter of how different merchants use Shopp. I think having the checkbox state remembered is win win and can't confuse anyone.
